contracts/coordinator/.solhintignore

```diff
@@ -0,0 +1,2 @@
+# solhint can't parse `abi.decode` syntax.
+contracts/src/MixinCoordinatorApprovalVerifier.sol
```

Comment:
I tried but solhint throws a parser error when it hits that contract. I'm a little confused about why this just started happening and with only certain forms of `abi.decode()`. I don't think it's just my setup because the same thing happened on CI (that's how I first noticed it).